src/index.js

```diff
@@ -336,7 +336,8 @@ function removeNodeFromHost(host, node) {
 }
 
 function removeSlotFromRoot(root, node) {
-  node.assignedNodes().forEach(slotNodeFromSlot);
+  const assignedNodes = Array.prototype.slice.call(node.assignedNodes());
```

Comment:
@nadiam84 does that work in all browsers? I think that might require `Array.from()` to be polyfilled.